config/locales/server.en.yml

```
@@ -18,6 +18,28 @@ en:
   akismet:
     delete_reason: "determined by %{performed_by} to be a spammer"
 
+  discourse_akismet:
+    reviewables:
+      actions:
+        confirm_spam:
+          title: "Confirm Spam"
+          description: "Delete the post and submit spam feedback to Akismet"
+          complete: "Post deleted and marked as spam"
+        not_spam:
+          title: "Not Spam"
+          description: "Mark as ham and recover the post if deleted"
```

```suggestion
          description: "Mark as not spam and recover the post if deleted"
```

I think there is a typo here.